src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -154,18 +176,17 @@ If you want to use the automatic `_routes.json` generation, but want to exclude
 
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 142 to 159]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->This should have a file name and also the editor frame.

`dist/_routes.json`, **i guess**<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/withastro/docs/5978?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->